src/net.h

```diff
@@ -378,6 +381,8 @@ class CConnman
     unsigned int nSendBufferMaxSize{0};
     unsigned int nReceiveFloodSize{0};
 
+    uint16_t m_default_listen_port;
```

Comment:
In commit "net: pass listen port into CConnman" (dcf461abdb9ad6c769aab072b7522d1e5c189b20)

Throughout this commit I'm confused why `default_listen_port` and `DefaultListenPort` are used instead of just `listen_port` and `ListenPort`. It seems like these names are referring to the actual, active listening port, not the network default port.